rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q4-2023_{context}"]
 === Q4 2023
 
+* **Configure custom security groups.** With the release of ROSA CLI (`rosa`) version 1.2.31, administrators can use the use the `rosa create cluster` and `rosa create machinepool` commands to configure a new cluster or a new machine pool with up to 5 additional custom security groups. Additionally, administrators can create a new machine pool with up to 5 additional custom security groups using OpenShift Cluster Manager. This allows administrators to have greater control over access to resources in new clusters and machine pools. For more information, see xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc#rosa-security-groups_prerequisites[Security groups].
```

Comment:
1. Remove extra "use the". 
2. using/by using as per IBM SG
3. "Allow" is a form of anthropomorphism. Could the statement be revised to something such as "By configuring custom security groups, an administrator can have greater..."?

![Screenshot from 2023-11-27 12-05-11](https://github.com/openshift/openshift-docs/assets/57954076/583894e8-bbd9-479f-a292-3ebce22094aa)

Reply:
Thanks @dfitzmau, good catch on the double words - going to blame my own butterfingers and unintentional vi commands for that one.